src/css/map.css

```diff
@@ -0,0 +1,114 @@
+.map-container {
+    text-align: center;
+}
+
+.map-row {
+    display: flex;
+    justify-content: center;
+}
+
+path {
```

Comment:
might wanna access a class before path... like `.map-container path {` so it doesn't affect every path on the page